### PR TITLE
Support Ubuntu 21.04

### PR DESCRIPTION
**Pull request by tetra12**
### Description

Solves #5361

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?

Built, deployed and launched an Advanced Zone on a single host with Ubuntu 21.04

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -22,7 +21,7 @@ Description: CloudStack server library
 
 Package: cloudstack-agent
 Architecture: all
-Depends: ${python:Depends}, ${python3:Depends}, openjdk-11-jre-headless | java11-runtime-headless | java11-runtime | openjdk-11-jre-headless | zulu-11, cloudstack-common (= ${source:Version}), lsb-base (>= 9), openssh-client, qemu-kvm (>= 2.5), libvirt-bin (>= 1.3) | libvirt-daemon-system (>= 3.0), iproute2, ebtables, vlan, ipset, python3-libvirt, ethtool, iptables, lsb-release, aria2, ufw, apparmor
+Depends: ${python:Depends}, ${python3:Depends}, openjdk-11-jre-headless | java11-runtime-headless | java11-runtime | openjdk-11-jre-headless | zulu-11, cloudstack-common (= ${source:Version}), lsb-base (>= 9), openssh-client, libvirt-bin (>= 1.3) | libvirt-daemon-system (>= 3.0), iproute2, ebtables, vlan, ipset, python3-libvirt, ethtool, iptables, lsb-release, aria2, ufw, apparmor
```
Comment (shwstppr) [inline debian/control:24]: @tetra12 with this change we might get support for Ubuntu 21.04 but won't it break build for earlier versions?

File: debian/control [hunk 2]
```diff
@@ -22,7 +21,7 @@ Description: CloudStack server library
 
 Package: cloudstack-agent
 Architecture: all
-Depends: ${python:Depends}, ${python3:Depends}, openjdk-11-jre-headless | java11-runtime-headless | java11-runtime | openjdk-11-jre-headless | zulu-11, cloudstack-common (= ${source:Version}), lsb-base (>= 9), openssh-client, qemu-kvm (>= 2.5), libvirt-bin (>= 1.3) | libvirt-daemon-system (>= 3.0), iproute2, ebtables, vlan, ipset, python3-libvirt, ethtool, iptables, lsb-release, aria2, ufw, apparmor
```
Comment (rhtyd) [inline debian/control:25]: You may want to replace qemu-kvm with qemu-system-x86

File: debian/control [hunk 3]
```diff
@@ -2,8 +2,7 @@ Source: cloudstack
 Section: libs
 Priority: extra
 Maintainer: Wido den Hollander <wido@widodh.nl>
-Build-Depends: debhelper (>= 9), openjdk-11-jdk | java11-sdk | java11-jdk | zulu-11, genisoimage,
- python-mysql.connector | python3-mysql.connector, maven (>= 3) | maven3, python (>= 2.7), python3 (>= 3), nodejs (>= 12), lsb-release, dh-systemd, python-setuptools
```
Comment (rhtyd) [inline debian/control:6]: Instead of removing build dependencies - you want to find equivalent package names version

File: debian/rules [hunk 4]
```diff
@@ -96,7 +96,7 @@ override_dh_auto_install:
 	# cloudstack-ui
 	mkdir $(DESTDIR)/$(SYSCONFDIR)/$(PACKAGE)/ui
 	mkdir -p $(DESTDIR)/usr/share/$(PACKAGE)-ui
-	cd ui && npm install && npm run build && cd ..
+	cd ui && npm config set unsafe-perm true && npm install && npm rebuild node-sass --force && npm run build && cd ..
```
Comment (rhtyd) [inline debian/rules:99]: @tetra12 are you using nodejs LTS?

**Comment by weizhouapache** [comment]
@tetra12 
can you revert the changes which are unrelated to the ubuntu 21.04 support ?

The only change we need seems to be
```
qemu-kvm (>= 2.5) | qemu-system-x86 (>= 4.2), 
```

**Comment by nvazquez** [comment]
Ping @tetra12 can you please address the open comments?

**Comment by tetra12** [comment]
Closing due to inactivity

**Comment by weizhouapache** [comment]
@tetra12 
I have created #5755 to support qemu-system-x86

**Pull request closed without merging**